### ISSUE TITLE
y predicted values given x

### ISSUE THREAD

**Issue by ast972** [8080labs/ppscore#71]
Good morning all,
I just installed this interesting package, and I find the PPS interesting and well motivated for feature selection. Given a vector y, and a vector x, you tell us whether x has 'explanatory power' relative to y, measured between 0,1. And viceversa. So given any attempt to write a causal model for y, we should somehow consider x.

You are fitting a model behind the scenes, that you choose from a menu of models (and if I understand correctly you focus on the decision tree class). But then, I wonder how can I see the 'predicted' values for y (what econometricians call yhat) given the variables x (what  econometricians call 'explanatory variable') for the given sample (y,x). Next would of course be the model 'yhat' for a fresh x, not in the sample.

Perhaps this is buried in the dict output but I couldnt find it. Could you help?

Thank you 

Angelo

**Comment by 8080labs** [comment]
Currently, yhat is not returned. What is the background of your request? In which situation do you want that?

**Comment by ast972** [comment]
indeed, when we estimate a correlation rho between two normalized random variables (x,y), we can approximate the conditional expectation of y given x as yhat = rho*x. this is useful in many applications, the most obvious is linear regression. Is there an equivalent in ppscore measure?

**Comment by 8080labs** [comment]
Understood. There is no equivalent available. If you want to get a prediction for a certain value, you might want to train your own decision tree(s).Sent from mobileOn 24 Jan 2023, at 21:37, 'ast972' via Admin ***@***.***> wrote:﻿
indeed, when we estimate a correlation rho between two normalized random variables (x,y), we can approximate the conditional expectation of y given x as yhat = rho*x. this is useful in many applications, the most obvious is linear regression. Is there an equivalent in ppscore measure?
